CHANGES.rst

```diff
@@ -132,6 +132,9 @@ Cubeviz
 Imviz
 ^^^^^
 
+- Fixes cropped image layer with WCS-linking without fast-approximation, mouseover display
```

Comment:
Not sure about hyphen but I should really defer to @ojustino et al.
```suggestion
- Fixes cropped image layer with WCS linking without fast-approximation, mouseover display
```

Reply:
Since I was summoned, I'd say you're right about no hyphen on these, @pllim.